man/aptly.1

```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "APTLY" "1" "March 2017" "" ""
+.TH "APTLY" "1" "2017-03-23" "" ""
```

Comment:
this seems to be generated with _broken_ ronn, `make man` should install correct `ronn` fork for you and run man page generation